### PR TITLE
Our project

### PR DESCRIPTION
**Pull request by Sayamistar**
Please review the login screen and register screen codes.

### CODE REVIEW

File: gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/LoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        txtForgotPassword.setMovementMethod(LinkMovementMethod.getInstance());
+    }
+    public void loginHandler(View view) {
+        String getUser = txtUser.getText().toString();
+        String getPass = txtPass.getText().toString();
+
+        if (getUser.equals("") || getPass.equals("")) {
+            Toast.makeText(getApplicationContext(),"Please enter all the values", Toast.LENGTH_LONG).show();
+        }else{
+            Boolean chkUser=DB.checkUserName(getUser);
```
Comment (rajeshbista) [inline gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/LoginActivity.java:68]: is password encrypted?

Comment (Sayamistar) [reply gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/LoginActivity.java:68]: We should hash-code the password in later updates.

Comment (suvayeti) [reply gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/LoginActivity.java:68]: For sure!

File: gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import android.os.Bundle;
+import android.os.Handler;
+import android.view.WindowManager;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+    private static int SPLASH_TIME=4000;
```
Comment (Sayamistar) [inline gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/MainActivity.java:15]: Isn't 4000 too long? Do we need that long?

Comment (Das69) [reply gym_app/app/src/main/java/ca/gbc/comp3074/gym_application/MainActivity.java:15]: The animation looks good at this time.

**Pull request merged**